admin_guide/configuring_authentication.adoc

```diff
@@ -464,7 +464,7 @@ the Apache VirtualHost:
     provider:
       apiVersion: v1
       kind: RequestHeaderIdentityProvider
-      clientCA: /etc/openshift/master/ca.crt
+      clientCA: /etc/openshift/master/proxy/ca.crt
```

Comment:
That CA is generated by `oadm create-api-client-config ...` in example 6. It's the same file as /etc/openshift/master/proxyca.crt which could be used instead but either would need to be the clientCA in the master identity configuration.

```
# md5sum proxyca.crt 
c2c915a9185f304f57f2ba1c8b414f72  proxyca.crt
# md5sum proxy/ca.crt 
c2c915a9185f304f57f2ba1c8b414f72  proxy/ca.crt
```

```
/etc/openshift/master/proxy
├── ca.crt
├── system:proxy.crt
├── system:proxy.key
└── system:proxy.kubeconfig
```